heron/cli/src/python/main.py

```diff
@@ -221,21 +223,14 @@ def main():
   args, unknown_args = parser.parse_known_args()
   command_line_args = vars(args)
 
-  try:
-    if command_line_args['verbose']:
-      opts.set_verbose()
-    if command_line_args['trace_execution']:
-      opts.set_trace_execution()
-  except:
-    pass
-
   # command to be execute
   command = command_line_args['subcommand']
 
   # file resources to be cleaned when exit
   files = []
 
-  if command != 'help' and command != 'version':
+  if command not in ('help', 'version'):
+    log.set_verbose(command_line_args)
```

Comment:
this is not the correct logic - only if the --verbose is provided, then enable the debug level option.